src/main/resources/db/migration/V1_36__add_long_description_to_intervention.sql

```diff
@@ -0,0 +1,6 @@
+alter table intervention
+    add column long_description text not null default '';
+
+update intervention
+    set long_description = description
+    where long_description = '';
```

Comment:
Not sure if no newline will cause an issue in circle. It usually fails for Kotlin files.